### PR TITLE
Remove ip from logging, require nickname

### PR DESCRIPTION
**Pull request by zhyatt**

### CODE REVIEW

File: nmonit.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def _filter_host(connect:str) -> str:
 
 def main(connect: str, slack: str, discord: str, nickname: str) -> None:
     if nickname != "":
-        address = f"{nickname} ({_filter_host(connect)})"
+        address = f"{nickname}"
     else:
         address = _filter_host(connect)
```
Comment (argakiig) [inline nmonit.py]: what about here as well? if a nickname is not provided it will still show the connection string host value

Comment (argakiig) [reply nmonit.py]: seems like perhaps the better thing to do would be to require nickname and enforce its usage instead

Comment (zhyatt) [reply nmonit.py]: Ah yeah, that makes sense. What is the best way to enforce that in this setup? Are you able to push that change here and I can review?

**Comment by argakiig** [comment]
pushed update, please verify working @zhyatt

**Comment by argakiig** [comment]
now requires nickname when passing flags during launch

**Review by argakiig — APPROVED** [review]

**Pull request merged**